<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    font-family: &quot;Inter&quot;, &quot;Segoe UI&quot;, sans-serif;
	font-size: 14px;
}
#centralwidget {
	border-radius: 8px;
	background-color: rgb(255, 255, 255);
}
#navFrame {
	margin: 0px;
	background-color: #f1f1f1;
	border-bottom-left-radius: 8px;

}
QLabel {
	font-family: &quot;Poppins&quot;, &quot;Segoe UI&quot;, sans-serif;
	font-size: 14px;
	font-weight: 400;
}
QLabel.title {
	font-size: 24px;
	font-weight: 600;
 }
QLabel.subtitle {
	font-size: 16px;
	font-weight: 500;
}
QLineEdit {
	border: 1px solid #404040;
	border-radius: 4px;
 	font-family: &quot;Open Sans&quot;, &quot;Segoe UI&quot;, sans-serif;
	font-size: 14px;
	padding: 2px;
}
QLineEdit:focus {
	border: 2px solid rgb(72, 72, 72);
}
QPlainTextEdit {
	border: 1px solid #404040;
	border-radius: 4px;
	padding: 8px;
	font-family: &quot;JetBrains Mono&quot;, &quot;Fira Code&quot;, &quot;Consolas&quot;, monospace;
	font-size: 13px;
	line-height: 1.4;       
 }
QPlainTextEdit:focus {
	border: 2px solid rgb(72, 72, 72);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="titleFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border:none;
	font-size: 12px;
	padding: 5px;
}
QPushButton:hover {
	background-color: rgba(141, 141, 141, 0.3);
    border-color: #3daee9;
}
#titleFrame {
	border-top-right-radius: 8px;
	border-top-left-radius: 8px;
	background-color: #f3f3f3;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1020</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="minimizeBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/nav/icons/minus-svgrepo-com.svg</normaloff>:/nav/icons/minus-svgrepo-com.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maximizeBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/nav/icons/square-svgrepo-com.svg</normaloff>:/nav/icons/square-svgrepo-com.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="styleSheet">
          <string notr="true">QPushButton::hover{
	background-color: rgb(255, 5, 59);
	border-top-right-radius: 8px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/nav/icons/close-mini-1522-svgrepo-com.svg</normaloff>:/nav/icons/close-mini-1522-svgrepo-com.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="navFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border:none;
	text-align: left;
	font-size: 12px;
	padding: 5px;
}
QPushButton:hover {
	border-radius:8px;
    background-color: #e1e1e1;
    border-color: #3daee9;
}
QPushButton:checked {
    background-color: #e1e1e1;
	border-radius:8px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="homeBtn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>Trang chủ</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/nav/icons/home-svgrepo-com.svg</normaloff>:/nav/icons/home-svgrepo-com.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="settingsBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>Cài đặt</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/nav/icons/settings-svgrepo-com.svg</normaloff>:/nav/icons/settings-svgrepo-com.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="contactBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>Liên hệ</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/nav/icons/support-svgrepo-com.svg</normaloff>:/nav/icons/support-svgrepo-com.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="helpBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>Trợ giúp</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/nav/icons/help-circle-svgrepo-com.svg</normaloff>:/nav/icons/help-circle-svgrepo-com.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QStackedWidget" name="main_stacked_widget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="home_page">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Trang chủ</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="tableView">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_selected">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;Đã chọn: &lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::RichText</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_success">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Thành công: &lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::RichText</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_failed">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Thất bại: &lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::RichText</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="settings_page">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="text">
                 <string>Cài đặt</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_5">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QFrame" name="frame_6">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>30</y>
                   <width>210</width>
                   <height>47</height>
                  </rect>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Đường dẫn:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="pathEdit"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QFrame" name="frame_7">
                 <property name="geometry">
                  <rect>
                   <x>250</x>
                   <y>30</y>
                   <width>151</width>
                   <height>47</height>
                  </rect>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>số luồng:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="threadSP"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QFrame" name="frame_8">
                 <property name="geometry">
                  <rect>
                   <x>580</x>
                   <y>30</y>
                   <width>151</width>
                   <height>47</height>
                  </rect>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Chọn:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="chooseCb">
                    <item>
                     <property name="text">
                      <string>1</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>2</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>3</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>4</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QFrame" name="frame_11">
                 <property name="geometry">
                  <rect>
                   <x>410</x>
                   <y>30</y>
                   <width>151</width>
                   <height>47</height>
                  </rect>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Độ trễ</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="delaySP"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QCheckBox" name="hidenTabCB">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>90</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Ẩn tab</string>
                 </property>
                </widget>
                <widget class="QCheckBox" name="delFileCB">
                 <property name="geometry">
                  <rect>
                   <x>150</x>
                   <y>90</y>
                   <width>151</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Xoá file sau khi xong</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="contact_page">
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>570</x>
                <y>150</y>
                <width>51</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Liên hệ</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="help_page">
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>610</x>
                <y>140</y>
                <width>51</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Hỗ trợ</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
